ci/test/04_install.sh

```
@@ -105,7 +105,7 @@
 
 if [ "$RUN_FUZZ_TESTS" = "true" ]; then
   export DIR_FUZZ_IN=${DIR_QA_ASSETS}/fuzz_seed_corpus/
-  if [ ! -d "$DIR_FUZZ_IN" ]; then
+  if [ \! -d "$DIR_FUZZ_IN" ]; then
```

Shellcheck (0.10.0)

[warning] Unknown binary operator.

(SC2057)